draftjs_exporter/wrapper_state.py

```diff
@@ -128,7 +128,16 @@ def element_for(self, block):
         options = Options.for_block(self.block_map, type_)
 
         # Make an element from the options specified in the block map.
-        elt = DOM.create_element(options.element[0], options.element[1])
+        tag = options.element[0]
+        attributes = options.element[1]
+
+        if (hasattr(tag, 'render')):
+            # If element wrapper is specified as object with 'render' method
+            attributes['data'] = block.get('data')
```

Comment:
Well, I don't know. As for me, the convention "if block has data, we output it to 'data' attribute" is not the worst, because it's pretty straightforward and predictable.

On the other hand, I don't like how rigid it is—if 'data' attribute is needed for other goals, there'll be no way to change attribute name, of something like this. What do you think?

We can rename it to something more unique/rare as well as HTML5 valid, e.g. 'data-draftjs-data'.